python/build_ceed_cffi.py

```diff
@@ -13,6 +13,27 @@
 ceed_version_ge = re.compile(r'\s+\(!?CEED_VERSION.*')
 
 
+# Checks to see if a c line is part of the lines we have to exclude (macros)
+def is_valid_line(line):
+    if (line.startswith("#") and not line.startswith("#include")):
+        return False
+    if (line.startswith("  CEED_QFUNCTION_ATTR")):
+        return False
+    if (line.startswith("  static")):
+        return False
+    if (line.endswith('\\\n')):
+        return False
```

Comment:
This first being needed means the second is not working, so we should fix the second

Reply:
```suggestion
    if (line.endswith('\\')):
        return False
```
This might be the fix, if the lines are split on `\n`

Reply:
It's so that the `static const char` declaration is able to be the last line of this macro:
```c++
#ifndef CEED_QFUNCTION_RUST
#define CEED_QFUNCTION_RUST(name)                                                                                            \
  CEED_QFUNCTION_ATTR int        name##_rs(void *ctx, const CeedInt Q, const CeedScalar *const *in, CeedScalar *const *out); \
  CEED_QFUNCTION_ATTR static int name(void *ctx, const CeedInt Q, const CeedScalar *const *in, CeedScalar *const *out) {     \
    return name##_rs(ctx, Q, in, out);                                                                                       \
  }                                                                                                                          \
  static const char name##_loc[] = __FILE__ ":" #name;
#endif
```